spyder/widgets/fileswitcher.py

```diff
@@ -669,17 +670,24 @@ def setup_file_list(self, filter_text, current_path):
                     text_item += " [{0:} {1:}]".format(self.line_count[index],
                                                        _("lines"))
                 if max_width > self.list.width():
-                    text_item += (u" &nbsp; <span style='color:{0:};"
+                    text_item += (u" &nbsp; <span style='float: left; color:{0:};"
                                   "font-size:{1:}pt'>{2:}"
                                   "</span>").format(self.PATH_FG_COLOR,
                                                     path_text_font_size,
                                                     short_paths[index])
                 else:
-                    text_item += (u" &nbsp; <span style='color:{0:};"
+                    text_item += (u" &nbsp; <span style='float: left; color:{0:};"
                                   "font-size:{1:}pt'>{2:}"
                                   "</span>").format(self.PATH_FG_COLOR,
                                                     path_text_font_size,
                                                     paths[index])
+                    title = self.widgets[index][1].get_plugin_title().split(' - ')[0]
+                    text_item += (u"<span style= 'float: right; color:{0:};"
```

Comment:
Remove the extra space after ``span style= ``; add space after ``color:`` for style consistency.